101-create-documentdb-account-ext/README.md

```diff
@@ -0,0 +1,14 @@
+# Create DocumentDB Account with a specified Consistency Policy
```

Comment:
ext means its "extended" or not basic, which the other one is. This one adds in consistency policy. Can rename this if you like.